application/client.js

```diff
@@ -152,7 +152,14 @@ module.exports = HttpGateway.extend({
                     try {
                         responseData = JSON.parse(responseText);
                     } catch (e) {
-                        responseData = responseText;
```

Comment:
Yeah, that sounds undesirable...So it sounds like your research shows we *cannot* access the native/automatically dispatched OPTIONS response? Is that correct?